### ISSUE TITLE
Unexpected hyphen for addresses in India with no postcode

### ISSUE THREAD

**Issue by mmmaction** [placemarkt/address-formatter-java#7]
Hi

I am using this great library.
For the following input values with country India and no `postcode` provided, the outcome is not as expected and contains an unwanted `hyphen`  (-> `Pune -`), see test below.

The following JUnit Test

```java
    @Test
    void test_AddressFormatter_noPostalCode_India() throws IOException {

        var formatter = new AddressFormatter(false, false);
        String json = "{country_code: 'IN',\n"
                + "road: 'Baner',\n"
                + "city: 'Pune',\n"
                + "state: 'Maharashtra',\n"
                + "country: 'India',}";
        String formatted = formatter.format(json);

        var expected = "Baner\nPune\nMaharashtra\nIndia\n";
        assertThat(formatted, equalTo(expected));
    }
```

Leads to
```bash
java.lang.AssertionError: 
Expected: "Baner\nPune\nMaharashtra\nIndia\n"
     but: was "Baner\nPune -\nMaharashtra\nIndia\n"
```

**Comment by rkoeze** [comment]
Thanks for reporting this @mmmaction and apologies for the delayed response.  I became a father recently and have been adjusting to a new set of priorities! I have more bandwidth now and will take a look.

**Comment by rkoeze** [comment]
@mmmaction this has been fixed in release [v0.0.12](https://github.com/placemarkt/address-formatter-java/releases/tag/v0.0.12). Thank you for the bug report!
